src/resolvers/teams.js

```diff
@@ -87,6 +106,10 @@ const teamsResolvers = {
       await team.save();
       return team;
     },
+    async inviteTeam(root, { emails, teamId }) {
+      sendTeamInvites(emails, teamId);
+      return true;
```

Comment:
maybe graphql does handle errors nicely internally, but it doesn't seem like this catches errors thrown 

you might want to wrap this in a try/catch, and subsequently might have to wrap `sendTeamInvites` in try/catch.